commands_to_load/Mod.py

```diff
@@ -82,5 +83,460 @@ def __init__(self, bot):
+        logger.info('[Mod lock()] minion role found: {}'.format(MINIONS_ROLE.id))
+
+        # Edit channel with new permissions
+        logger.info('[Mod lock()] editing {} permissions'.format(channel))
+        await channel.set_permissions(ctx.guild.default_role, send_messages=False)
+        await channel.set_permissions(MINIONS_ROLE, send_messages=True)
+
+        # Message channel to notify the status
+        logger.info('[Mod lock()] lock message sent to {}'.format(channel))
+        eObj = await em(ctx, description='This channel has been locked until further notice.', author=ctx.author.display_name, avatar=ctx.author.avatar_url, footer='Moderator action')
+        if eObj is not False:
+            await ctx.send(embed=eObj)
+
+        # Inform council of the locked channel
+        logger.info('[Mod lock()] council informed of locked channel')
+        council = discord.utils.get(ctx.guild.channels, name='council')
+        eObj = await em(ctx, description='{} locked {}'.format(ctx.message.author, channel.mention), footer='Moderator action')
+        if eObj is not False:
+            await council.send(embed=eObj)
+
+    @commands.command()
+    async def unlock(self, ctx):
+        # Unlocks a channel if its locked
+        # Verify channel is locked 
+        ## How? => check @everyone for send_messages
+        
+        logger.info('[Mod unlock()] unlock function detected by user' + str(ctx.message.author))
+        await ctx.message.delete()
+        logger.info('[Mod unlock()] invoking command deleted')
+
+        if not ctx.message.author in discord.utils.get(ctx.guild.roles, name="Minions").members:
+            logger.info('[Mod unlock()] unathorized command attempt detected. Being handled.')
+            await self.rekt(ctx)
+            return
+
+        channel = ctx.channel
+        perms = channel.overwrites
+        ow = list(perms[0]) # assumes @everyone is always the fifrst thign in teh perms tuple
+
+        if ow[1].send_messages == True or ow[1].send_messages == None: 
+            # Not locked
+            eObj = await em(ctx, description='You can\'t unlock what isn\'t locked.\n-Richard Stallman\'s Fart', footer='Command error')
+            if eObj is not False:
+                await ctx.send(embed=eObj, delete_after=5.0)
+            return
+
+        # If here then the channel is locked and we can proceed 
+        
+        # Get the Minions role
+        MINIONS_ROLE = discord.utils.get(ctx.guild.roles, name='Minions')
+        logger.info('[Mod unlock()] minion role found: {}'.format(MINIONS_ROLE.id))
+
+        # Set the permissions
+        logger.info('[Mod unlock()] editing {} permissions'.format(channel))
+        await channel.set_permissions(ctx.guild.default_role, send_messages=None)
+        await channel.set_permissions(MINIONS_ROLE, overwrite=None)
+
+        # Tell the channel
+        logger.info('[Mod unlock()] unlock message sent to {}'.format(channel))

```

Comment:
isnt the lock message technically sent on 530, with that being verified on line 531? so this should be below line 532